trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.split.first(10).join(' ')
```

Comment:
- [ ] This piece of code seems to be a little faulty since it returns the first 10 words, not characters. It needs slight altering so you split the string at each character and join it without a space. Something like
``` ruby
'string'.split('').first(10).join('')
```
Kindly fix this decorator before your next submission

Reply:
Thank you for the idea